### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -14924,6 +14927,7 @@ mutches->matches
 muti->multi
 muticast->multicast
 mutices->mutexes
+mutiindex->multiindex, multi index,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: `Multiindex` should be hyphenated unless it's a special word in a programming language, in which case it probably shouldn't be first.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -20093,8 +20126,8 @@ sate->state, sate,
 satelite->satellite
 satelites->satellites
 satelitte->satellite
-Saterday->Saturday
-Saterdays->Saturdays
+saterday->saturday
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Saturday is a proper noun so I think the RHS should start with a capital.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: b9992e9

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: b9992e9

**Comment by luzpaz** [comment]
Fixed and ready for merge if there is no more feedback

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -17125,7 +17138,9 @@ postcondtions->postconditions
 Postdam->Potsdam
 postgress->postgresql
 postgressql->PostgreSQL
+postgrsql->postgresql
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Capitalise this and the top one to match the middle one.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -20009,6 +20041,7 @@ rouine->routine
 rouines->routines
 round-robbin->round-robin
 roundign->rounding
+roundtriped->roundtripped
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Also suggest the space and hyphen separated versions

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -20153,6 +20186,8 @@ sattelite->satellite
 sattelites->satellites
 sattellite->satellite
 sattellites->satellites
+satuaday->saturday
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Again capitalise.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -22910,6 +22963,8 @@ tranport->transport
 transacion->transaction
 transaction-spacific->transaction-specific
 transactoin->transaction
+transaition->translation, transition,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Also suggest transaction

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: d71d521

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: d71d521 + added more variations

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: e54d28f

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: e54d28f

**Comment by luzpaz** [comment]
I made requested revisions. I jumped the gun and pushed the branch prematurely, hence I pushed the PR twice just now, apologies.

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -20077,9 +20109,16 @@ rouine->routine
 rouines->routines
 round-robbin->round-robin
 roundign->rounding
+roundtriped->roundtripped, round-tripped, round tripped,
+roundtriping->roundtripping, round-tripping, round tripping,
+roundtripp->roundtrip
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can we add the other suggestions to this one too please?

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: specifically?

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: `roundtrip, round-trip, round trip,`

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Done, 2909386

**Pull request merged**